packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx

```diff
@@ -25,6 +26,10 @@ export type LayoutMenuProps = {
    * Breakpoint above which the menu button displays as a full sidebar
    */
   breakpoint?: 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+  /**
+   * An array of section items containing the title, slug, href, and onClick, each of which become an additional link on top of the accordion section.
```

Comment:
Nit: you can skip describing what TypeScript already indicates. Not a blocker 😄 

```suggestion
   * An array of section items, each of which become an additional link on top of the accordion section.
```